web/src/types/index.ts

```diff
@@ -1,28 +1,75 @@
 export type App = {
```

Comment:
okay I could be wrong here but I THINK that if you use interfaces instead of types in this file then you don't need to export them. then you also don't need to import them anywhere and can use them everywhere within your code

Reply:
That makes sense. I've been following the examples in bullet proof react and [they're just exporting the types](https://github.com/alan2207/bulletproof-react/blob/master/src/types/index.ts). 

I think having global types is cleaner and we should do that long term. I think we have to go through a re-naming process for some of this stuff first.

Like we have a `Dashboard` component and `DashboardResponse` type. But we also have an `App` type that maybe should be called `AppResponse`. I'm not sure what kind of naming conventions we should use yet.